test/rulesets/Base/array.jl

```diff
@@ -35,3 +35,9 @@ end
     test_rrule(fill, 44.0, 4; check_inferred=false)
     test_rrule(fill, 2.0, (3, 3, 3) ⊢ DoesNotExist())
 end
+
+@testset "repeat" begin
+    test_rrule(repeat, randn(5), 3)
+    test_rrule(repeat, randn(3,3), 2)
+    test_rrule(repeat, randn(5,4,3); fkwargs=(inner=(2,2,1), outer=(1,1,3)))
```

Comment:
Should we add a few more tests? E.g. both matrix and vectors for both 1 and two indices. And similarly a few edge cases for the inner outer one?

Perhaps we could also test for `Float64[]` or `fill(4.0)`, both zero dimensional arrays

Reply:
What do you want the result to be for the empty array? Repeat just returns a zero-dim array for that.